provider/provider.go

```diff
@@ -31,6 +31,12 @@ func KeycloakProvider(client *keycloak.KeycloakClient) *schema.Provider {
 		ResourcesMap: map[string]*schema.Resource{
 			"keycloak_realm":                                             resourceKeycloakRealm(),
 			"keycloak_realm_events":                                      resourceKeycloakRealmEvents(),
+			"keycloak_realm_key_aes_generated":                           resourceKeycloakRealmKeystoreAesGenerated(),
+			"keycloak_realm_key_ecdsa_generated":                         resourceKeycloakRealmKeystoreEcdsaGenerated(),
+			"keycloak_realm_key_hmac_generated":                          resourceKeycloakRealmKeystoreHmacGenerated(),
+			"keycloak_realm_key_java_keystore":                           resourceKeycloakRealmKeystoreJavaKeystore(),
+			"keycloak_realm_key_rsa":                                     resourceKeycloakRealmKeystoreRsa(),
+			"keycloak_realm_key_rsa_generated":                           resourceKeycloakRealmKeystoreRsaGenerated(),
```

Comment:
what did you intend for these resources to be called?

for example, you use `keycloak_realm_key_rsa_generated` here and in the headers for the documentation, but the file names that were added are `realm_keystore_rsa_generated.md` and `resource_keycloak_realm_keystore_rsa_generated.go`.

to me, I think `keystore` makes more sense than `key`, since the Keycloak GUI refers to these as Keystores.  regardless, we'll want to make some changes so the resource names are consistent.